contactsummary/CRM/Contactsummary/Utils/Absences.php

```diff
@@ -57,7 +57,7 @@ private static function getAbsenceDuration($absenceTypeNames = array(), $periodI
       ON (a.id = a1.source_record_id AND a1.activity_type_id = %1 AND a1.status_id = %2)
 
       WHERE
-        a.id IN (" . implode(',', $absenceTypeIds) . ")
+        a.id IN (" . (count($absenceTypeIds) > 0 ? implode(',', $absenceTypeIds) : 'NULL') . ")
```

Comment:
How did you get this line to throw an error? You didn't have any absence type defined?